css/style.css

```diff
@@ -16,4 +15,149 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*HEADER STYLES*/
+
+#karma-logo {
+  width: 25px;
+  margin-left: 50px;
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+nav {
+  float: right;
+  margin-top: 50px;
+  margin-right: 50px;
+}
+
+nav a {
+  margin-right: 40px;
+  text-decoration: none;
+  color: gray;
+}
+
+#karma-meet {
+  font-weight: bold;
+}
+nav a:hover {
+  color: #c05326;
+}
+
+/*CONTENT STYLES*/
+h1 {
+  text-align: center;
+  color: white;
+  font-size: 75px;
+  font-weight: lighter;
+}
+
+h2 {
+  text-align: center;
+  color: white;
+  font-weight: lighter;
+  font-size: 40px;
+  margin-top: 0;
+}
+
+button {
+  align-content: center;
```

Comment:
`align-content` is a property used to adjust flex/grid cells content horizontally/vertically
in order for this to work the element must also be declared as `display: flex` or `display: grid`
but the `button` is not currently a flex/grid container
maybe i am wrong, but i am assuming you are trying to center the button text with this?